src/constants/surveys.ts

```diff
@@ -0,0 +1,4 @@
+export const REPORT_PROBLEM_ACCESSIBILITY_TEAM_SURVEY =
```

Comment:
I'm on the fence about this file because these links are only used once. Another reason is that this file has a pretty narrow focus. If we could widen it's scope, it might be more useful.

Reply:
I changed the filename to `externalUrls`, which I imagine could get utilized for other constants as well.